0x48piraj/app/styles/app.css

```diff
@@ -0,0 +1,150 @@
+@import url(https://fonts.googleapis.com/css?family=Bungee);
```

Comment:
Are you using all fonts? also, import should not be used. It should be avoided. Read more about it here: https://www.giftofspeed.com/avoid-using-css-import/ and https://www.stevesouders.com/blog/2009/04/09/dont-use-import/